curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md

```diff
@@ -22,9 +22,9 @@ Newlines and other whitespace may be ignored unless contained within a quoted st
 
 Handling escaped quotes inside a string is optional; thus "`(foo"bar)`" may be treated as a string "`foo"bar`", or as an error.
 
-For this, the reader need not recognize "<code>\\</code>" for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
+For this, the reader need not recognize `\` for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
 
-Note that with the exception of "`()"`" ("<code>\\</code>" if escaping is supported) and whitespace there are no special characters. Anything else is allowed without quotes.
+Note that with the exception of `()"` (`\` if escaping is supported) and whitespace, there are no special characters. Anything else is allowed without quotes.
```

Comment:
I think the other double quote should be removed.  I think the double quotes were left over when they used to surround the open and close parentheses.
```suggestion
Note that with the exception of `()` (`\` if escaping is supported) and whitespace, there are no special characters. Anything else is allowed without quotes.
```